tests/unit-core/CMakeLists.txt

```diff
@@ -23,8 +23,26 @@ if(${FEATURE_PROFILE} STREQUAL "${CMAKE_SOURCE_DIR}/jerry-core/profiles/minimal.
   message(FATAL_ERROR "minimal profile isn't supported in unit-core")
 endif()
 
-# Unit tests main modules
-file(GLOB SOURCE_UNIT_TEST_MAIN_MODULES *.c)
+# Unit tests main modules that exercise the API:
+if(NOT EMSCRIPTEN_SIMULATED_JERRY_API)
+  file(GLOB SOURCE_UNIT_TEST_MAIN_API_MODULES api/*.c)
+else()
+  set(SOURCE_UNIT_TEST_MAIN_API_MODULES
+      api/test-api.c
+      api/test-context-data.c
+      api/test-has-property.c
+      )
```

Comment:
I would prefer to use the same glob and then remove/exclude test files that are unsupported by the emx build.

Reply:
make sense